drivers/bus/acpi_new/acpi.inf

```diff
@@ -0,0 +1,131 @@
+
+; ACPI.INF
```

Comment:
Is this a copy of our existing acpi.inf file?

Reply:
yes

Reply:
Because this one is going to have a bunch of stuff added to it?